core/src/main/scala/zio/jdbc/Sql.scala

```diff
@@ -71,14 +71,14 @@ final class Sql[+A](
 
     val params = paramsBuilder.result()
 
-    val paramsString = if (params.length > 0) {
+    val paramsString = if (params.nonEmpty) {
       ", " + params.mkString(", ")
     } else ""
 
     s"Sql(${sql.result()}${paramsString})"
   }
 
-  def values[B](
+  private def values[B](
```

Comment:
Was it intentional to make this method `private`?

Reply:
@jdegoes it was but since the last PR you just merged, it is no longer necessary will change.